interface/reports/appointments_report.php

```diff
@@ -425,22 +490,52 @@ function refreshme() {
     $apptdate_list = array(); // same as above for the appt details
     $totalAppontments = count($appointments);
 
+    $cntr = 1; // column labels above start at 1
     foreach ($appointments as $appointment) {
+        $cntr++;
         array_push($pid_list, $appointment['pid']);
         array_push($apptdate_list, $appointment['pc_eventDate']);
         $patient_id = $appointment['pid'];
         $docname  = $appointment['ulname'] . ', ' . $appointment['ufname'] . ' ' . $appointment['umname'];
 
         $errmsg  = "";
         $pc_apptstatus = $appointment['pc_apptstatus'];
+        if ($_POST['form_csvexport']) {
+            if (empty($appointment['phone_home'])) {
+                $phone_home_csv = $appointment['phone_cell'];
+            } else {
+                $phone_home_csv = $appointment['phone_home'];
+            }
 
-        ?>
+            $phone_home_csv = csvEscape($phone_home_csv, $surroundQuotes = false);
```

Comment:
2nd parameter above change to `false` rather than `$surroundQuotes = false` (other calls below look good)